api.js

```diff
@@ -898,7 +898,9 @@ API.prototype.reference = function(options) {
   assert(options,         "Options is required");
   assert(options.baseUrl, "A 'baseUrl' must be provided");
   var reference = {
-    version:            '0.2.0',
+    version:            0,
+    '$schema':          'http://schemas.taskcluster.net/base/v1/' +
+                        'api-reference.json',
```

Comment:
Sure, I'll add it... I think it's optional to add the "#", but yes, it is to indicate that it's the root.